docs/hugo/content/guide/authentication/_index.md

```diff
@@ -12,12 +12,13 @@ There are two key topics surrounding authentication in Azure Service Operator: T
 
 ## Credential type
 
-Azure Service Operator supports four different styles of authentication today.
+Azure Service Operator supports five different styles of authentication today.
 
 1. [Recommended for production] [Azure Workload Identity]( {{< relref "credential-format#managed-identity-via-workload-identity" >}} ) (OIDC + Managed Identity or Service Principal)
 2. [Service Principal using a Client Secret]( {{< relref "credential-format#service-principal-using-a-client-secret" >}} )
 3. [Service Principal using a Client Certificate]( {{< relref "credential-format#service-principal-using-a-client-certificate" >}} )
```

Comment:
Realized you also need to pass this new setting through to the pod:
See v2/config/manager/manager_image_patch.yaml and v2/charts/azure-service-operator/templates/apps_v1_deployment_azureserviceoperator-controller-manager.yaml

As discussed, we don't need any entry in Helm values.yaml to actually fill it out, but I assume that having these defined for you automatically in the deployment is desired for your use-case?